internal/services/bot/bot_channel_direct_line_speech_resource.go

```diff
@@ -191,6 +204,11 @@ func resourceBotChannelDirectLineSpeechUpdate(d *pluginsdk.ResourceData, meta in
 		Kind:     botservice.KindBot,
 	}
 
+	if v, ok := d.GetOk("cognitive_account_id"); ok {
```

Comment:
This should also be using d.HasChanges since it's in the update.

Reply:
Updated